src/components/metrics/modal/index.js

```diff
@@ -17,6 +18,18 @@ const percentages = (values, perc) => {
   return perc.map(e => { return {percentage: e, value: percentile(values, values.length, e)} })
 }
 
+const SampleList = ({samples}) => (
+  <ul className='SampleList'>
+    {_.map(samples, (element, index) => {
+      return (
+        <li key={index}>
+          <div key={index}>{element}</div>
```

Comment:
I believe React generally does for lists, I copied this from the data field where this was required.